docs/source/gbd2017_models/concept_models/vivarium_conic_lsff/concept_model.rst

```diff
@@ -98,6 +98,188 @@ Interventions
 Vitamin A Fortification
 ~~~~~~~~~~~~~~~~~~~~~~~
 
+Effect Size
+^^^^^^^^^^^
+
+**Research Considerations**
+
+In this model, the vitamin A fortification intervention affects the 
+**prevalence of vitamin A deficiency**. The effect size for this intervention 
+was obtained from a Cochrane review performed by Hombali et al. (2019) on the 
+fortification of staple foods with vitamin A for vitamin A deficiency. 
+Notably, the relative risk for vitamin A foritification on vitamin A 
+deficiency from this review only included data from two randomized controlled 
+trials and the authors of the review assessed the certainty of the evidence to 
+be "very low" (Hombali et al. 2019). The relative risk of vitamin A deficiency 
+prevalence among the population exposed to vitamin A fortification relative to 
+the population unexposed to vitamin A fortification from this review was 
```

Comment:
The relative risk of vitamin A deficiency prevalence among an intervention population exposed to vitamin A fortified staple foods relative to a control population given the same staple foods not fortified with vitamin A....

Reply:
Right! Updated :)